app/plugin.go

```diff
@@ -429,10 +440,16 @@ func (a *App) ShutDownPlugins() {
 	a.PluginEnv = nil
 }
 
+func getKeyHash(key string) string {
+	hash := sha256.New()
```

Comment:
No need for a crypto hash in this final implementation.

Reply:
I think it depends. It's extremely important that there's never a collision. I'm much more comfortable making that guarantee with SHA256 than I am with FNV. In my plugins, I would prefer the marginal performance hit (which I'm about to benchmark out of curiosity).

Reply:
Right and FNV has an extremely low probablity of collisions. It's just easier to find them if you want to. Curious on the perf difference please post.

Reply:
64 character string:

```
BenchmarkSHA256-8   	 2000000	       740 ns/op
BenchmarkFNV128-8   	 3000000	       461 ns/op
```

128 character string:

```
BenchmarkSHA256-8   	 2000000	       889 ns/op
BenchmarkFNV128-8   	 2000000	       809 ns/op
```

With larger strings, FNV is actually slower.

Reply:
FNV is pretty low, but it's not lower enough that collisions won't unexpectedly happen. 32-bit FNV-1a has some really simple ones ("costarring" and "liquid"). I bet I could find some 128-bit FNV-1a collisions relatively easily too.

Reply:
Huh. I expected an order of magnitude difference at least. It's faster, but not that much faster. Well forget it then. Doesn't really matter. Do whatever you want @jwilander

Reply:
If it's not going to significantly impact perf then let's just stick with SHA256